tests/unit/test_add_ssh_key.py

```diff
@@ -0,0 +1,32 @@
+from __future__ import annotations
+
+import jubilant
+from tests.unit import mocks
+
+
+def test_add_ssh_key_single(run: mocks.Run):
```

Comment:
This is in `test_add_ssh_key.py`, so shorten name (consistent with other unit test names too):
```suggestion
def test_single(run: mocks.Run):
```